### PR TITLE
Fasih/DPROD-3471/Add lazy loading attribute in fast payment section logos

### PR DESCRIPTION
**Pull request by fasihali-deriv**
Changes:

-  Removed lazy loading attribute from fast payment section logos

## Type of change

-   [x] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/components/hooks/use-is-in-viewport.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+function useIsInViewport(ref: MutableRefObject<HTMLDivElement>) {
+    const [isIntersecting, setIsIntersecting] = useState(false)
+    const observer = useRef<IntersectionObserver>()
+
+    useEffect(() => {
+        observer.current = new IntersectionObserver(([entry]) =>
+            setIsIntersecting(entry.isIntersecting),
+        )
+
+        observer.current.observe(ref.current)
```
Comment (mitra-deriv) [inline src/components/hooks/use-is-in-viewport.tsx:12]:
```suggestion
        observer.current?.observe(ref.current)
```

**Review by habib-deriv — APPROVED** [review]
LGTM

File: src/components/hooks/use-is-in-viewport.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+        return () => observer.current.disconnect()
+    }, [ref.current, observer.current])
```
Comment (mitra-deriv) [inline src/components/hooks/use-is-in-viewport.tsx:15]: observer.current needs to be a dependency ? does it change?

File: src/components/hooks/use-is-in-viewport.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    const observer = useRef<IntersectionObserver>()
+
+    useEffect(() => {
+        observer.current = new IntersectionObserver(([entry]) =>
+            setIsIntersecting(entry.isIntersecting),
+        )
+
+        observer.current.observe(ref.current)
+
+        return () => observer.current.disconnect()
```
Comment (mitra-deriv) [inline src/components/hooks/use-is-in-viewport.tsx:14]:
```suggestion
        return () => observer.current?.disconnect()
```

**Pull request closed without merging**